src/test/java/org/apache/maven/shared/filtering/MultiDelimiterInterpolatorFilterReaderLineEndingTest.java

```diff
@@ -121,4 +123,25 @@ public void testAtDollarExpression()
 
         assertEquals( "  url=\"jdbc:oracle:thin:@DB_SERVER:DB_PORT:DB_NAME\"", IOUtil.toString( reader ) );
     }
+    
+    @Test
+    public void testDefaultValue()
```

Comment:
What about adding tests to cover syntax error cases? Or empty string as default...